products/securityscanner/terraform.yaml

```diff
@@ -18,6 +18,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "scan_config_basic"
         primary_resource_id: "scan-config"
+        min_version: beta
```

Comment:
Why is this in Terraform and not in api.yaml?

Reply:
This is an example block for a specific terraform test. The resource is beta only so the example should only be generated for beta